subsys/net/lib/lwm2m/lwm2m_obj_firmware_pull.c

```diff
@@ -16,7 +16,8 @@ LOG_MODULE_REGISTER(LOG_MODULE_NAME);
 #include "lwm2m_pull_context.h"
 #include "lwm2m_engine.h"
 
-static void set_update_result_from_error(int error_code)
+static void set_update_result_from_error(struct firmware_pull_context *context,
+                                         int error_code)
 {
 	if (!error_code) {
```

Comment:
Woudl be good to use `ARG_UNUSED()` on `context` here.

Reply:
The function is now a bit misleading as setting update result from error when there is no error suggests something else.

Reply:
Hmmm @jarlamsa I understand but since it's a static function of this file do you think this is a problem? I think it's pretty readable that if no error is present then we report success. Maybe we can make the name more generic? `set_update_result`? I'm open to suggestions :)

@rlubos Added `ARG_UNUSED()`

Reply:
`set_update_result` would be good name for this

Reply:
fixed